spring-cloud-gcp-autoconfigure/src/main/java/com/google/cloud/spring/autoconfigure/kms/GcpKmsProperties.java

```diff
@@ -31,6 +31,9 @@ public class GcpKmsProperties implements CredentialsSupplier {
 
   /** Overrides the GCP Project ID specified in the Core module. */
   private String projectId;
+  private String universeDomain;
```

Comment:
I guess Spring would convert `universeDomain` to `universe-domain` automatically, so we don't have to specify the exact name of the property?

Reply:
Correct